Started GET "/category" for ::1 at 2020-08-19 01:20:35 -0400
  [1m[35m (15.6ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.3.2) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.3.2) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category" for ::1 at 2020-08-19 01:20:43 -0400
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.3.2) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.3.2) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category" for ::1 at 2020-08-19 01:20:45 -0400
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.3.2) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.3.2) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (0.9ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (1.0ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (1.1ms)[0m  [1m[35mCREATE TABLE "schema_migrations" ("version" varchar NOT NULL PRIMARY KEY)[0m
  [1m[35m (2.0ms)[0m  [1m[35mCREATE TABLE "ar_internal_metadata" ("key" varchar NOT NULL PRIMARY KEY, "value" varchar, "created_at" datetime(6) NOT NULL, "updated_at" datetime(6) NOT NULL)[0m
  [1m[35m (0.2ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Migrating to CreateCategories (20200819051502)
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  [1m[35m (0.6ms)[0m  [1m[35mCREATE TABLE "categories" ("id" integer PRIMARY KEY AUTOINCREMENT NOT NULL, "title" varchar, "description" text, "created_by" bigint, "created_at" datetime(6) NOT NULL, "updated_at" datetime(6) NOT NULL)[0m
  [1m[36mprimary::SchemaMigration Create (0.2ms)[0m  [1m[32mINSERT INTO "schema_migrations" ("version") VALUES (?)[0m  [["version", "20200819051502"]]
  [1m[35m (1.1ms)[0m  [1m[36mcommit transaction[0m
  [1m[36mActiveRecord::InternalMetadata Load (0.2ms)[0m  [1m[34mSELECT "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = ? LIMIT ?[0m  [["key", "environment"], ["LIMIT", 1]]
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  [1m[36mActiveRecord::InternalMetadata Create (0.4ms)[0m  [1m[32mINSERT INTO "ar_internal_metadata" ("key", "value", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["key", "environment"], ["value", "development"], ["created_at", "2020-08-19 05:21:37.324527"], ["updated_at", "2020-08-19 05:21:37.324527"]]
  [1m[35m (1.8ms)[0m  [1m[36mcommit transaction[0m
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Started GET "/category" for ::1 at 2020-08-19 01:21:52 -0400
  [1m[35m (2.6ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.3ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [GET] "/category"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category" for ::1 at 2020-08-19 01:21:55 -0400
  
ActionController::RoutingError (No route matches [GET] "/category"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category" for ::1 at 2020-08-19 01:23:38 -0400
  [1m[35m (2.0ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by CategoryController#index as */*
  Parameters: {"title"=>"Learn Python", "created_by"=>1, "description"=>"Programming Book", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Exists? (0.2ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
Completed 200 OK in 2ms (Views: 0.3ms | ActiveRecord: 0.6ms | Allocations: 880)


Started POST "/category" for ::1 at 2020-08-19 01:23:43 -0400
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "created_by"=>1, "description"=>"Programming Book", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 200 OK in 118ms (Views: 0.1ms | ActiveRecord: 0.4ms | Allocations: 6281)


Started POST "/category" for ::1 at 2020-08-19 01:27:05 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "created_by"=>1, "description"=>"Programming Book", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 18ms (ActiveRecord: 0.9ms | Allocations: 3750)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:27:44 -0400
Error occurred while parsing request parameters.
Contents:

{
	"title": "Learn Python",
	"created_by": 1,
    "description": Programming Book
}
  
ActionDispatch::Http::Parameters::ParseError (783: unexpected token at '{
	"title": "Learn Python",
	"created_by": 1,
    "description": Programming Book
}'):
  
actionpack (6.0.3.2) lib/action_dispatch/http/parameters.rb:114:in `rescue in parse_formatted_parameters'
actionpack (6.0.3.2) lib/action_dispatch/http/parameters.rb:110:in `parse_formatted_parameters'
actionpack (6.0.3.2) lib/action_dispatch/http/request.rb:389:in `block in POST'
rack (2.2.3) lib/rack/request.rb:69:in `fetch'
rack (2.2.3) lib/rack/request.rb:69:in `fetch_header'
actionpack (6.0.3.2) lib/action_dispatch/http/request.rb:388:in `POST'
actionpack (6.0.3.2) lib/action_dispatch/http/parameters.rb:55:in `parameters'
actionpack (6.0.3.2) lib/action_controller/metal/params_wrapper.rb:278:in `_wrapper_enabled?'
actionpack (6.0.3.2) lib/action_controller/metal/params_wrapper.rb:244:in `process_action'
activerecord (6.0.3.2) lib/active_record/railties/controller_runtime.rb:27:in `process_action'
actionpack (6.0.3.2) lib/abstract_controller/base.rb:136:in `process'
actionpack (6.0.3.2) lib/action_controller/metal.rb:190:in `dispatch'
actionpack (6.0.3.2) lib/action_controller/metal.rb:254:in `dispatch'
actionpack (6.0.3.2) lib/action_dispatch/routing/route_set.rb:50:in `dispatch'
actionpack (6.0.3.2) lib/action_dispatch/routing/route_set.rb:33:in `serve'
actionpack (6.0.3.2) lib/action_dispatch/journey/router.rb:49:in `block in serve'
actionpack (6.0.3.2) lib/action_dispatch/journey/router.rb:32:in `each'
actionpack (6.0.3.2) lib/action_dispatch/journey/router.rb:32:in `serve'
actionpack (6.0.3.2) lib/action_dispatch/routing/route_set.rb:834:in `call'
rack (2.2.3) lib/rack/etag.rb:27:in `call'
rack (2.2.3) lib/rack/conditional_get.rb:40:in `call'
rack (2.2.3) lib/rack/head.rb:12:in `call'
activerecord (6.0.3.2) lib/active_record/migration.rb:567:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Error occurred while parsing request parameters.
Contents:

{
	"title": "Learn Python",
	"created_by": 1,
    "description": Programming Book
}
Started POST "/category" for ::1 at 2020-08-19 01:28:35 -0400
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "created_by"=>1, "description"=>"Programming Book", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 2ms (ActiveRecord: 0.0ms | Allocations: 734)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started GET "/category" for ::1 at 2020-08-19 01:28:47 -0400
Processing by CategoryController#index as */*
  Parameters: {"title"=>"Learn Python", "created_by"=>1, "description"=>"Programming Book", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Exists? (0.3ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
Completed 200 OK in 2ms (Views: 0.2ms | ActiveRecord: 0.3ms | Allocations: 729)


Started POST "/category" for ::1 at 2020-08-19 01:31:31 -0400
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "created_by"=>1, "description"=>"Programming Book", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 13ms (ActiveRecord: 0.0ms | Allocations: 734)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:33:12 -0400
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "created_by"=>1, "description"=>"Programming Book", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 3ms (ActiveRecord: 0.0ms | Allocations: 724)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:34:22 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 31ms (ActiveRecord: 1.3ms | Allocations: 3760)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:37:29 -0400
  [1m[35m (0.8ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 24ms (ActiveRecord: 0.0ms | Allocations: 724)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:37:52 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[36mCategory Create (0.6ms)[0m  [1m[32mINSERT INTO "categories" ("title", "created_by", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["title", "Learn Python"], ["created_by", 1], ["created_at", "2020-08-19 05:37:53.051172"], ["updated_at", "2020-08-19 05:37:53.051172"]]
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[35m (1.6ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
Completed 500 Internal Server Error in 126ms (ActiveRecord: 3.0ms | Allocations: 36042)


  
NoMethodError (undefined method `json_response' for #<CategoryController:0x00007fed8b28bb78>):
  
app/controllers/category_controller.rb:26:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:39:52 -0400
  [1m[35m (1.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[36mCategory Create (1.4ms)[0m  [1m[32mINSERT INTO "categories" ("title", "created_by", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["title", "Learn Python"], ["created_by", 1], ["created_at", "2020-08-19 05:39:52.938704"], ["updated_at", "2020-08-19 05:39:52.938704"]]
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[35m (3.1ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
Completed 500 Internal Server Error in 187ms (ActiveRecord: 7.5ms | Allocations: 38197)


  
NoMethodError (undefined method `json_response' for #<CategoryController:0x00007fed87cdb758>):
  
app/controllers/category_controller.rb:26:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:41:35 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 20ms (ActiveRecord: 1.8ms | Allocations: 3760)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:41:54 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[36mCategory Create (0.6ms)[0m  [1m[32mINSERT INTO "categories" ("title", "created_by", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["title", "Learn Python"], ["created_by", 1], ["created_at", "2020-08-19 05:41:54.223588"], ["updated_at", "2020-08-19 05:41:54.223588"]]
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[35m (1.7ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
Completed 500 Internal Server Error in 90ms (ActiveRecord: 3.7ms | Allocations: 35905)


  
NoMethodError (undefined method `json_response' for #<CategoryController:0x00007fed889d7d58>):
  
app/controllers/category_controller.rb:26:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:42:45 -0400
  [1m[35m (0.2ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 18ms (ActiveRecord: 0.8ms | Allocations: 3744)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:42:57 -0400
  [1m[35m (0.2ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[36mCategory Create (0.4ms)[0m  [1m[32mINSERT INTO "categories" ("title", "created_by", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["title", "Learn Python"], ["created_by", 1], ["created_at", "2020-08-19 05:42:57.926769"], ["updated_at", "2020-08-19 05:42:57.926769"]]
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[35m (1.6ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
Completed 500 Internal Server Error in 131ms (ActiveRecord: 2.9ms | Allocations: 35907)


  
NoMethodError (undefined method `json_response' for #<CategoryController:0x00007fed88987df8>):
  
app/controllers/category_controller.rb:26:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:44:24 -0400
  [1m[35m (0.9ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[36mCategory Create (3.4ms)[0m  [1m[32mINSERT INTO "categories" ("title", "created_by", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["title", "Learn Python"], ["created_by", 1], ["created_at", "2020-08-19 05:44:25.016802"], ["updated_at", "2020-08-19 05:44:25.016802"]]
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[35m (2.5ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
Completed 500 Internal Server Error in 149ms (ActiveRecord: 12.0ms | Allocations: 36415)


  
NoMethodError (undefined method `json_response' for #<CategoryController:0x00007f980a456cb8>):
  
app/controllers/category_controller.rb:26:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:44:48 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 21ms (ActiveRecord: 1.7ms | Allocations: 4851)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:46:14 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
Completed 422 Unprocessable Entity in 9ms (ActiveRecord: 0.7ms | Allocations: 3721)


  
ActiveRecord::RecordInvalid (Validation failed: Description can't be blank):
  
app/controllers/category_controller.rb:25:in `create'
Started POST "/category" for ::1 at 2020-08-19 01:46:32 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
[31mUnpermitted parameters: :description, :category[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[36mCategory Create (1.5ms)[0m  [1m[32mINSERT INTO "categories" ("title", "created_by", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["title", "Learn Python"], ["created_by", 1], ["created_at", "2020-08-19 05:46:32.198530"], ["updated_at", "2020-08-19 05:46:32.198530"]]
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[35m (1.5ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
Completed 201 Created in 22ms (Views: 1.0ms | ActiveRecord: 4.1ms | Allocations: 5436)


Started PUT "/category" for ::1 at 2020-08-19 01:47:00 -0400
  
ActionController::RoutingError (No route matches [PUT] "/category"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started DELETE "/category" for ::1 at 2020-08-19 01:47:06 -0400
  
ActionController::RoutingError (No route matches [DELETE] "/category"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category" for ::1 at 2020-08-19 01:47:41 -0400
Processing by CategoryController#index as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Exists? (0.4ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories"[0m
  ↳ app/controllers/category_controller.rb:15:in `index'
Completed 200 OK in 7ms (Views: 3.9ms | ActiveRecord: 0.6ms | Allocations: 2346)


Started DELETE "/category/1" for ::1 at 2020-08-19 01:47:52 -0400
Processing by CategoryController#destroy as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "id"=>"1", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[36mCategory Destroy (0.4ms)[0m  [1m[31mDELETE FROM "categories" WHERE "categories"."id" = ?[0m  [["id", 1]]
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[35m (1.2ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
Completed 200 OK in 23ms (Views: 0.3ms | ActiveRecord: 1.8ms | Allocations: 3992)


Started GET "/category/1" for ::1 at 2020-08-19 18:27:21 -0400
  [1m[35m (7.6ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.5ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by CategoryController#show as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "id"=>"1", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Load (0.5ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
Completed 404 Not Found in 6ms (ActiveRecord: 1.4ms | Allocations: 1224)


  
ActiveRecord::RecordNotFound (Couldn't find Category with 'id'=1):
  
app/controllers/category_controller.rb:54:in `set_category'
Started GET "/category" for ::1 at 2020-08-19 18:27:27 -0400
Processing by CategoryController#index as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Exists? (0.2ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories"[0m
  ↳ app/controllers/category_controller.rb:15:in `index'
Completed 200 OK in 38ms (Views: 35.2ms | ActiveRecord: 0.4ms | Allocations: 5288)


Started DELETE "/category2" for ::1 at 2020-08-19 18:27:43 -0400
  
ActionController::RoutingError (No route matches [DELETE] "/category2"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started DELETE "/category2" for ::1 at 2020-08-19 18:27:46 -0400
  
ActionController::RoutingError (No route matches [DELETE] "/category2"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/category2" for ::1 at 2020-08-19 18:27:57 -0400
  
ActionController::RoutingError (No route matches [PUT] "/category2"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started DELETE "/category/2" for ::1 at 2020-08-19 18:28:06 -0400
Processing by CategoryController#destroy as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "id"=>"2", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 2], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[36mCategory Destroy (0.6ms)[0m  [1m[31mDELETE FROM "categories" WHERE "categories"."id" = ?[0m  [["id", 2]]
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[35m (0.9ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
Completed 200 OK in 7ms (Views: 0.2ms | ActiveRecord: 1.8ms | Allocations: 2505)


Started DELETE "/category/3" for ::1 at 2020-08-19 18:28:09 -0400
Processing by CategoryController#destroy as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "id"=>"3", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Load (0.4ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 3], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[36mCategory Destroy (0.4ms)[0m  [1m[31mDELETE FROM "categories" WHERE "categories"."id" = ?[0m  [["id", 3]]
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[35m (0.9ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
Completed 200 OK in 6ms (Views: 0.2ms | ActiveRecord: 1.7ms | Allocations: 2465)


Started DELETE "/category/4" for ::1 at 2020-08-19 18:28:12 -0400
Processing by CategoryController#destroy as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "id"=>"4", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Load (0.1ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 4], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[36mCategory Destroy (0.5ms)[0m  [1m[31mDELETE FROM "categories" WHERE "categories"."id" = ?[0m  [["id", 4]]
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[35m (0.8ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
Completed 200 OK in 5ms (Views: 0.2ms | ActiveRecord: 1.6ms | Allocations: 2480)


Started DELETE "/category/5" for ::1 at 2020-08-19 18:28:16 -0400
Processing by CategoryController#destroy as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "id"=>"5", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 5], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[36mCategory Destroy (0.5ms)[0m  [1m[31mDELETE FROM "categories" WHERE "categories"."id" = ?[0m  [["id", 5]]
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[35m (0.9ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
Completed 200 OK in 6ms (Views: 0.2ms | ActiveRecord: 1.7ms | Allocations: 2481)


Started DELETE "/category/6" for ::1 at 2020-08-19 18:28:20 -0400
Processing by CategoryController#destroy as */*
  Parameters: {"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1, "id"=>"6", "category"=>{"title"=>"Learn Python", "description"=>"Programming Book", "created_by"=>1}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 6], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[36mCategory Destroy (0.6ms)[0m  [1m[31mDELETE FROM "categories" WHERE "categories"."id" = ?[0m  [["id", 6]]
  ↳ app/controllers/category_controller.rb:42:in `destroy'
  [1m[35m (3.6ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:42:in `destroy'
Completed 200 OK in 12ms (Views: 0.3ms | ActiveRecord: 4.5ms | Allocations: 2480)


Started GET "/category/" for ::1 at 2020-08-19 18:28:41 -0400
Processing by CategoryController#index as */*
  [1m[36mCategory Exists? (0.3ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
Completed 200 OK in 2ms (Views: 0.2ms | ActiveRecord: 0.3ms | Allocations: 669)


  [1m[35m (1.5ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.7ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Migrating to CreateRecipes (20200819223613)
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  [1m[35m (1.1ms)[0m  [1m[35mCREATE TABLE "recipes" ("id" integer PRIMARY KEY AUTOINCREMENT NOT NULL, "name" varchar, "ingredients" varchar, "directions" text, "notes" varchar, "tags" varchar, "created_at" datetime(6) NOT NULL, "updated_at" datetime(6) NOT NULL)[0m
  [1m[36mprimary::SchemaMigration Create (0.6ms)[0m  [1m[32mINSERT INTO "schema_migrations" ("version") VALUES (?)[0m  [["version", "20200819223613"]]
  [1m[35m (1.5ms)[0m  [1m[36mcommit transaction[0m
  [1m[36mActiveRecord::InternalMetadata Load (0.8ms)[0m  [1m[34mSELECT "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = ? LIMIT ?[0m  [["key", "environment"], ["LIMIT", 1]]
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.2ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Started GET "/recipe/" for ::1 at 2020-08-19 18:49:15 -0400
  [1m[35m (4.3ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipe" for ::1 at 2020-08-19 18:49:47 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for ::1 at 2020-08-19 18:49:51 -0400
Processing by Rails::WelcomeController#index as */*
  Rendering /Users/andreheslop/.rvm/gems/ruby-2.7.0/gems/railties-6.0.3.2/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /Users/andreheslop/.rvm/gems/ruby-2.7.0/gems/railties-6.0.3.2/lib/rails/templates/rails/welcome/index.html.erb (Duration: 45.0ms | Allocations: 409)
Completed 200 OK in 109ms (Views: 87.9ms | ActiveRecord: 0.0ms | Allocations: 2295)


Started GET "/recipe" for ::1 at 2020-08-19 18:50:36 -0400
  [1m[35m (1.5ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.7ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipe" for ::1 at 2020-08-19 18:50:44 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipe" for ::1 at 2020-08-19 18:52:00 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipes" for ::1 at 2020-08-19 18:53:21 -0400
  [1m[35m (1.4ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.2ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by RecipesController#index as */*
  [1m[36mRecipe Exists? (0.2ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:10:in `index'
Completed 200 OK in 66ms (Views: 0.9ms | ActiveRecord: 0.7ms | Allocations: 2521)


Started GET "/recipe" for ::1 at 2020-08-19 18:55:13 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipes" for ::1 at 2020-08-19 18:55:15 -0400
Processing by RecipesController#index as */*
  [1m[36mRecipe Exists? (0.2ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:10:in `index'
Completed 200 OK in 11ms (Views: 0.3ms | ActiveRecord: 0.8ms | Allocations: 1840)


Started GET "/category" for ::1 at 2020-08-19 18:55:25 -0400
Processing by CategoryController#index as */*
  [1m[36mCategory Exists? (2.5ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
Completed 200 OK in 55ms (Views: 0.2ms | ActiveRecord: 2.8ms | Allocations: 2291)


Started GET "/recipes" for ::1 at 2020-08-19 18:55:43 -0400
Processing by RecipesController#index as */*
  [1m[36mRecipe Exists? (0.3ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:10:in `index'
Completed 200 OK in 2ms (Views: 0.2ms | ActiveRecord: 0.3ms | Allocations: 668)


Started PUT "/recipes" for ::1 at 2020-08-19 19:05:19 -0400
  [1m[35m (0.5ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipes" for ::1 at 2020-08-19 19:05:42 -0400
Processing by RecipesController#index as */*
  [1m[36mRecipe Exists? (0.9ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:10:in `index'
Completed 200 OK in 6ms (Views: 0.7ms | ActiveRecord: 0.9ms | Allocations: 670)


Started PUT "/recipe" for ::1 at 2020-08-19 19:05:55 -0400
  
ActionController::RoutingError (No route matches [PUT] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/recipes" for ::1 at 2020-08-19 19:06:17 -0400
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/recipes" for ::1 at 2020-08-19 19:06:52 -0400
  [1m[35m (1.4ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.5ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/recipes" for ::1 at 2020-08-19 19:09:11 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/recipes" for ::1 at 2020-08-19 19:10:03 -0400
  [1m[35m (0.2ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/recipes" for ::1 at 2020-08-19 19:11:53 -0400
  [1m[35m (6.3ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.2ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/recipes" for ::1 at 2020-08-19 19:14:28 -0400
  [1m[35m (1.3ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/recipes" for ::1 at 2020-08-19 19:15:50 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/recipes" for ::1 at 2020-08-19 19:16:49 -0400
  [1m[35m (3.0ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.5ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [PUT] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category" for ::1 at 2020-08-19 19:16:54 -0400
Processing by CategoryController#index as */*
  [1m[36mCategory Exists? (1.4ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
Completed 200 OK in 14ms (Views: 0.2ms | ActiveRecord: 1.8ms | Allocations: 2460)


Started PUT "/category/recipes" for ::1 at 2020-08-19 20:07:37 -0400
  [1m[35m (0.4ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by CategoryController#update as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "id"=>"recipes", "category"=>{}}
  [1m[36mCategory Load (0.8ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", nil], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
Completed 404 Not Found in 76ms (ActiveRecord: 7.3ms | Allocations: 2231)


  
ActiveRecord::RecordNotFound (Couldn't find Category with 'id'=recipes):
  
app/controllers/category_controller.rb:54:in `set_category'
Started PUT "/category/recipe" for ::1 at 2020-08-19 20:07:53 -0400
Processing by CategoryController#update as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "id"=>"recipe", "category"=>{}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", nil], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
Completed 404 Not Found in 2ms (ActiveRecord: 0.2ms | Allocations: 613)


  
ActiveRecord::RecordNotFound (Couldn't find Category with 'id'=recipe):
  
app/controllers/category_controller.rb:54:in `set_category'
Started PUT "/category/1/recipe" for ::1 at 2020-08-19 20:08:18 -0400
  
ActionController::RoutingError (No route matches [PUT] "/category/1/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/category/1/recipes" for ::1 at 2020-08-19 20:08:25 -0400
  
ActionController::RoutingError (No route matches [PUT] "/category/1/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started PUT "/category" for ::1 at 2020-08-19 20:09:32 -0400
  
ActionController::RoutingError (No route matches [PUT] "/category"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category" for ::1 at 2020-08-19 20:10:09 -0400
Processing by CategoryController#index as */*
  [1m[36mCategory Exists? (0.6ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
Completed 200 OK in 3ms (Views: 0.5ms | ActiveRecord: 0.6ms | Allocations: 671)


Started PUT "/category" for ::1 at 2020-08-19 20:10:17 -0400
  
ActionController::RoutingError (No route matches [PUT] "/category"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/recipes" for ::1 at 2020-08-19 20:10:52 -0400
  
ActionController::RoutingError (No route matches [POST] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/recipe" for ::1 at 2020-08-19 20:10:56 -0400
  
ActionController::RoutingError (No route matches [POST] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/category" for ::1 at 2020-08-19 20:11:03 -0400
Processing by CategoryController#create as */*
  Parameters: {"title"=>"test", "created_by"=>1, "category"=>{"title"=>"test", "created_by"=>1}}
[31mUnpermitted parameter: :category[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[36mCategory Create (3.3ms)[0m  [1m[32mINSERT INTO "categories" ("title", "created_by", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["title", "test"], ["created_by", 1], ["created_at", "2020-08-20 00:11:03.214404"], ["updated_at", "2020-08-20 00:11:03.214404"]]
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[35m (2.2ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
Completed 201 Created in 24ms (Views: 0.7ms | ActiveRecord: 5.6ms | Allocations: 5795)


Started POST "/category/7/recipe" for ::1 at 2020-08-19 20:12:10 -0400
  
ActionController::RoutingError (No route matches [POST] "/category/7/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:12:15 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 500 Internal Server Error in 109ms (ActiveRecord: 1.2ms | Allocations: 30608)


  
NoMethodError (undefined method `set_category' for #<RecipesController:0x00007fb2de834788>
Did you mean?  set_trace_func):
  
activesupport (6.0.3.2) lib/active_support/callbacks.rb:428:in `block in make_lambda'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:200:in `block (2 levels) in halting'
actionpack (6.0.3.2) lib/abstract_controller/callbacks.rb:34:in `block (2 levels) in <module:Callbacks>'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:201:in `block in halting'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:513:in `block in invoke_before'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:513:in `each'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:513:in `invoke_before'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:134:in `run_callbacks'
actionpack (6.0.3.2) lib/abstract_controller/callbacks.rb:41:in `process_action'
actionpack (6.0.3.2) lib/action_controller/metal/rescue.rb:22:in `process_action'
actionpack (6.0.3.2) lib/action_controller/metal/instrumentation.rb:33:in `block in process_action'
activesupport (6.0.3.2) lib/active_support/notifications.rb:180:in `block in instrument'
activesupport (6.0.3.2) lib/active_support/notifications/instrumenter.rb:24:in `instrument'
activesupport (6.0.3.2) lib/active_support/notifications.rb:180:in `instrument'
actionpack (6.0.3.2) lib/action_controller/metal/instrumentation.rb:32:in `process_action'
actionpack (6.0.3.2) lib/action_controller/metal/params_wrapper.rb:245:in `process_action'
activerecord (6.0.3.2) lib/active_record/railties/controller_runtime.rb:27:in `process_action'
actionpack (6.0.3.2) lib/abstract_controller/base.rb:136:in `process'
actionpack (6.0.3.2) lib/action_controller/metal.rb:190:in `dispatch'
actionpack (6.0.3.2) lib/action_controller/metal.rb:254:in `dispatch'
actionpack (6.0.3.2) lib/action_dispatch/routing/route_set.rb:50:in `dispatch'
actionpack (6.0.3.2) lib/action_dispatch/routing/route_set.rb:33:in `serve'
actionpack (6.0.3.2) lib/action_dispatch/journey/router.rb:49:in `block in serve'
actionpack (6.0.3.2) lib/action_dispatch/journey/router.rb:32:in `each'
actionpack (6.0.3.2) lib/action_dispatch/journey/router.rb:32:in `serve'
actionpack (6.0.3.2) lib/action_dispatch/routing/route_set.rb:834:in `call'
rack (2.2.3) lib/rack/etag.rb:27:in `call'
rack (2.2.3) lib/rack/conditional_get.rb:40:in `call'
rack (2.2.3) lib/rack/head.rb:12:in `call'
activerecord (6.0.3.2) lib/active_record/migration.rb:567:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.3.2) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.3.2) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/actionable_exceptions.rb:17:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:13:01 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 6ms (ActiveRecord: 1.8ms | Allocations: 1325)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:60:in `set_category'
Started POST "/category/1/recipes" for ::1 at 2020-08-19 20:13:34 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"1", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 0ms (ActiveRecord: 0.0ms | Allocations: 135)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:60:in `set_category'
Started GET "/category" for ::1 at 2020-08-19 20:13:40 -0400
Processing by CategoryController#index as */*
  [1m[36mCategory Exists? (2.3ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories"[0m
  ↳ app/controllers/category_controller.rb:15:in `index'
Completed 200 OK in 14ms (Views: 9.9ms | ActiveRecord: 2.6ms | Allocations: 4462)


Started POST "/category/1/recipes" for ::1 at 2020-08-19 20:13:53 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"1", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 0ms (ActiveRecord: 0.0ms | Allocations: 134)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:60:in `set_category'
Started POST "/category/1/recipe" for ::1 at 2020-08-19 20:13:57 -0400
  
ActionController::RoutingError (No route matches [POST] "/category/1/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/category/1/recipes" for ::1 at 2020-08-19 20:14:00 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"1", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 0ms (ActiveRecord: 0.0ms | Allocations: 134)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:60:in `set_category'
Started POST "/category/1/recipes" for ::1 at 2020-08-19 20:16:08 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"1", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 7ms (ActiveRecord: 1.1ms | Allocations: 1308)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/1/recipes" for ::1 at 2020-08-19 20:17:44 -0400
  [1m[35m (0.5ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"1", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 4ms (ActiveRecord: 1.7ms | Allocations: 1308)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:17:48 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 1ms (ActiveRecord: 0.0ms | Allocations: 134)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:17:52 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 0ms (ActiveRecord: 0.0ms | Allocations: 134)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/categories/7/recipes" for ::1 at 2020-08-19 20:18:42 -0400
  
ActionController::RoutingError (No route matches [POST] "/categories/7/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:18:46 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 1ms (ActiveRecord: 0.0ms | Allocations: 134)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/4/recipes" for ::1 at 2020-08-19 20:18:58 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"4", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 0ms (ActiveRecord: 0.0ms | Allocations: 134)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:19:05 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 0ms (ActiveRecord: 0.0ms | Allocations: 134)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/7" for ::1 at 2020-08-19 20:20:57 -0400
  
ActionController::RoutingError (No route matches [POST] "/category/7"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category/7" for ::1 at 2020-08-19 20:21:03 -0400
Processing by CategoryController#show as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "id"=>"7", "category"=>{}}
  [1m[36mCategory Load (1.4ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
Completed 200 OK in 53ms (Views: 8.1ms | ActiveRecord: 1.4ms | Allocations: 3703)


Started GET "/category/7/recipe" for ::1 at 2020-08-19 20:21:16 -0400
  
ActionController::RoutingError (No route matches [GET] "/category/7/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category/7/recipes" for ::1 at 2020-08-19 20:21:19 -0400
Processing by RecipesController#index as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 1ms (ActiveRecord: 0.0ms | Allocations: 134)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started GET "/category/7/recipes" for ::1 at 2020-08-19 20:22:08 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#index as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.6ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
  [1m[36mRecipe Exists? (0.3ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:11:in `index'
Completed 200 OK in 37ms (Views: 0.4ms | ActiveRecord: 2.0ms | Allocations: 5357)


Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:22:14 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameters: :category_id, :recipe[0m
Completed 500 Internal Server Error in 178ms (ActiveRecord: 0.5ms | Allocations: 8705)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:23:00 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 7ms (ActiveRecord: 1.3ms | Allocations: 1308)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:24:43 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.9ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameters: :category_id, :recipe[0m
Completed 500 Internal Server Error in 25ms (ActiveRecord: 2.0ms | Allocations: 9062)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/categories7/recipes" for ::1 at 2020-08-19 20:26:13 -0400
  
ActionController::RoutingError (No route matches [POST] "/categories7/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/category7/recipes" for ::1 at 2020-08-19 20:26:17 -0400
  
ActionController::RoutingError (No route matches [POST] "/category7/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:26:23 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameters: :category_id, :recipe[0m
Completed 500 Internal Server Error in 3ms (ActiveRecord: 0.2ms | Allocations: 979)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:27:30 -0400
  [1m[35m (0.0ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (1.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameters: :category_id, :recipe[0m
Completed 500 Internal Server Error in 32ms (ActiveRecord: 3.4ms | Allocations: 9061)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:28:20 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.4ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 25ms (ActiveRecord: 2.6ms | Allocations: 8907)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:28:42 -0400
  [1m[35m (0.2ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.1ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 30ms (ActiveRecord: 1.9ms | Allocations: 8943)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:29:21 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.4ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameters: :category_id, :recipe[0m
Completed 500 Internal Server Error in 34ms (ActiveRecord: 1.9ms | Allocations: 9062)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:29:37 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 39ms (ActiveRecord: 2.1ms | Allocations: 8926)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:31:36 -0400
  [1m[35m (0.2ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 3ms (ActiveRecord: 0.2ms | Allocations: 862)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:32:21 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 27ms (ActiveRecord: 2.1ms | Allocations: 8943)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:32:34 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.6ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 59ms (ActiveRecord: 3.0ms | Allocations: 8926)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:34:03 -0400
  [1m[35m (2.8ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.2ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 69ms (ActiveRecord: 2.1ms | Allocations: 11060)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:39:34 -0400
  [1m[35m (0.2ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.1ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 167ms (ActiveRecord: 2.3ms | Allocations: 8953)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:42:21 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 2ms (ActiveRecord: 0.2ms | Allocations: 879)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:43:42 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.1ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 20ms (ActiveRecord: 1.6ms | Allocations: 8942)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-19 20:44:56 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 7ms (ActiveRecord: 0.3ms | Allocations: 877)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:34:59 -0400
  [1m[35m (1.7ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 52ms (ActiveRecord: 2.1ms | Allocations: 1874)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:36:56 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.7ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 57ms (ActiveRecord: 2.4ms | Allocations: 10510)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:37:34 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 16ms (ActiveRecord: 2.1ms | Allocations: 1313)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:38:13 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (9.8ms)[0m  [1m[34mSELECT "categories".* FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 119ms (ActiveRecord: 49.3ms | Allocations: 9000)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:38:45 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 404 Not Found in 15ms (ActiveRecord: 9.5ms | Allocations: 1314)


  
ActiveRecord::RecordNotFound (Couldn't find Category without an ID):
  
app/controllers/recipes_controller.rb:61:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:40:27 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.6ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" IS NULL LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:61:in `set_category'
Completed 500 Internal Server Error in 6ms (ActiveRecord: 1.3ms | Allocations: 2372)


  
NoMethodError (undefined method `recipes' for nil:NilClass):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:42:20 -0400
  [1m[35m (0.2ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" IS NULL LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:59:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 10ms (ActiveRecord: 1.1ms | Allocations: 2470)


  
NoMethodError (undefined method `create!' for nil:NilClass):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:42:41 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" IS NULL LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:59:in `set_category'
Completed 500 Internal Server Error in 27ms (ActiveRecord: 4.1ms | Allocations: 2357)


  
NoMethodError (undefined method `recipes' for nil:NilClass):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:43:07 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:59:in `set_category'
[31mUnpermitted parameter: :recipe[0m
Completed 500 Internal Server Error in 214ms (ActiveRecord: 76.8ms | Allocations: 8960)


  
ActiveModel::UnknownAttributeError (unknown attribute 'category_id' for Recipe.):
  
app/controllers/recipes_controller.rb:26:in `create'
  [1m[35m (1.2ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (1.4ms)[0m  [1m[35mCREATE TABLE "schema_migrations" ("version" varchar NOT NULL PRIMARY KEY)[0m
  [1m[35m (1.6ms)[0m  [1m[35mCREATE TABLE "ar_internal_metadata" ("key" varchar NOT NULL PRIMARY KEY, "value" varchar, "created_at" datetime(6) NOT NULL, "updated_at" datetime(6) NOT NULL)[0m
  [1m[35m (0.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Migrating to CreateCategories (20200819051502)
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  [1m[35m (0.4ms)[0m  [1m[35mCREATE TABLE "categories" ("id" integer PRIMARY KEY AUTOINCREMENT NOT NULL, "title" varchar, "description" varchar, "created_by" bigint, "created_at" datetime(6) NOT NULL, "updated_at" datetime(6) NOT NULL)[0m
  [1m[36mprimary::SchemaMigration Create (0.2ms)[0m  [1m[32mINSERT INTO "schema_migrations" ("version") VALUES (?)[0m  [["version", "20200819051502"]]
  [1m[35m (1.0ms)[0m  [1m[36mcommit transaction[0m
Migrating to CreateRecipes (20200819223613)
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  [1m[35m (0.7ms)[0m  [1m[35mCREATE TABLE "recipes" ("id" integer PRIMARY KEY AUTOINCREMENT NOT NULL, "name" varchar, "ingredients" varchar, "directions" text, "notes" varchar, "tags" varchar, "category_id" integer NOT NULL, "created_at" datetime(6) NOT NULL, "updated_at" datetime(6) NOT NULL, CONSTRAINT "fk_rails_22f4e84158"
FOREIGN KEY ("category_id")
  REFERENCES "categories" ("id")
)[0m
  [1m[35m (0.1ms)[0m  [1m[35mCREATE  INDEX "index_recipes_on_category_id" ON "recipes" ("category_id")[0m
  [1m[36mprimary::SchemaMigration Create (0.2ms)[0m  [1m[32mINSERT INTO "schema_migrations" ("version") VALUES (?)[0m  [["version", "20200819223613"]]
  [1m[35m (0.9ms)[0m  [1m[36mcommit transaction[0m
  [1m[36mActiveRecord::InternalMetadata Load (0.2ms)[0m  [1m[34mSELECT "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = ? LIMIT ?[0m  [["key", "environment"], ["LIMIT", 1]]
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  [1m[36mActiveRecord::InternalMetadata Create (0.4ms)[0m  [1m[32mINSERT INTO "ar_internal_metadata" ("key", "value", "created_at", "updated_at") VALUES (?, ?, ?, ?)[0m  [["key", "environment"], ["value", "development"], ["created_at", "2020-08-20 11:49:41.490063"], ["updated_at", "2020-08-20 11:49:41.490063"]]
  [1m[35m (1.4ms)[0m  [1m[36mcommit transaction[0m
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:49:58 -0400
  [1m[35m (5.8ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:59:in `set_category'
Completed 500 Internal Server Error in 32ms (ActiveRecord: 3.1ms | Allocations: 2678)


  
NoMethodError (undefined method `recipes' for nil:NilClass):
  
app/controllers/recipes_controller.rb:26:in `create'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:50:47 -0400
  [1m[35m (0.4ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 500 Internal Server Error in 165ms (ActiveRecord: 1.2ms | Allocations: 60307)


  
NameError (undefined local variable or method `category_id' for #<RecipesController:0x00007fd6e426b958>
Did you mean?  category_url):
  
app/controllers/recipes_controller.rb:59:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:51:08 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
Completed 500 Internal Server Error in 3ms (ActiveRecord: 0.6ms | Allocations: 1177)


  
ArgumentError (Unsupported argument type: category_id (Symbol)):
  
app/controllers/recipes_controller.rb:59:in `set_category'
Started POST "/category/7/recipes" for ::1 at 2020-08-20 07:52:41 -0400
  [1m[35m (0.2ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"7", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (1.4ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 7], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:59:in `set_category'
Completed 404 Not Found in 38ms (ActiveRecord: 3.9ms | Allocations: 2181)


  
ActiveRecord::RecordNotFound (Couldn't find Category with 'id'=7):
  
app/controllers/recipes_controller.rb:59:in `set_category'
Started POST "/category" for ::1 at 2020-08-20 07:54:05 -0400
Processing by CategoryController#create as */*
  Parameters: {"title"=>"Test", "description"=>"Test", "created_by"=>1, "category"=>{"title"=>"Test", "description"=>"Test", "created_by"=>1}}
[31mUnpermitted parameter: :category[0m
  [1m[35m (0.3ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[36mCategory Create (0.8ms)[0m  [1m[32mINSERT INTO "categories" ("title", "description", "created_by", "created_at", "updated_at") VALUES (?, ?, ?, ?, ?)[0m  [["title", "Test"], ["description", "Test"], ["created_by", 1], ["created_at", "2020-08-20 11:54:05.455352"], ["updated_at", "2020-08-20 11:54:05.455352"]]
  ↳ app/controllers/category_controller.rb:25:in `create'
  [1m[35m (1.9ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/category_controller.rb:25:in `create'
Completed 201 Created in 37ms (Views: 2.9ms | ActiveRecord: 2.9ms | Allocations: 5896)


Started POST "/category/1/recipes" for ::1 at 2020-08-20 07:54:13 -0400
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"1", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:59:in `set_category'
[31mUnpermitted parameter: :recipe[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/recipes_controller.rb:26:in `create'
  [1m[36mRecipe Create (0.9ms)[0m  [1m[32mINSERT INTO "recipes" ("name", "ingredients", "directions", "notes", "tags", "category_id", "created_at", "updated_at") VALUES (?, ?, ?, ?, ?, ?, ?, ?)[0m  [["name", "test"], ["ingredients", "test"], ["directions", "test"], ["notes", "test"], ["tags", "test"], ["category_id", 1], ["created_at", "2020-08-20 11:54:13.628417"], ["updated_at", "2020-08-20 11:54:13.628417"]]
  ↳ app/controllers/recipes_controller.rb:26:in `create'
  [1m[35m (3.3ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/recipes_controller.rb:26:in `create'
Completed 500 Internal Server Error in 282ms (ActiveRecord: 5.1ms | Allocations: 14421)


  
NoMethodError (undefined method `name' for nil:NilClass):
  
app/controllers/recipes_controller.rb:27:in `create'
Started POST "/category/1/recipes" for ::1 at 2020-08-20 07:56:59 -0400
  [1m[35m (0.3ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"1", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
[31mUnpermitted parameter: :recipe[0m
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/recipes_controller.rb:26:in `create'
  [1m[36mRecipe Create (0.5ms)[0m  [1m[32mINSERT INTO "recipes" ("name", "ingredients", "directions", "notes", "tags", "category_id", "created_at", "updated_at") VALUES (?, ?, ?, ?, ?, ?, ?, ?)[0m  [["name", "test"], ["ingredients", "test"], ["directions", "test"], ["notes", "test"], ["tags", "test"], ["category_id", 1], ["created_at", "2020-08-20 11:56:59.771699"], ["updated_at", "2020-08-20 11:56:59.771699"]]
  ↳ app/controllers/recipes_controller.rb:26:in `create'
  [1m[35m (1.4ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/recipes_controller.rb:26:in `create'
Completed 201 Created in 67ms (Views: 0.3ms | ActiveRecord: 8.9ms | Allocations: 12109)


Started GET "/category/7/recipe" for ::1 at 2020-08-20 07:57:36 -0400
  
ActionController::RoutingError (No route matches [GET] "/category/7/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipe" for ::1 at 2020-08-20 07:57:52 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipes" for ::1 at 2020-08-20 07:57:55 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipes" for ::1 at 2020-08-20 07:57:59 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipe" for ::1 at 2020-08-20 07:58:21 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipe" for ::1 at 2020-08-20 07:59:14 -0400
  [1m[35m (10.8ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (5.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipes" for ::1 at 2020-08-20 07:59:19 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipes" for ::1 at 2020-08-20 07:59:25 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category" for ::1 at 2020-08-20 08:01:39 -0400
Processing by CategoryController#index as */*
  [1m[36mCategory Exists? (0.6ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories"[0m
  ↳ app/controllers/category_controller.rb:15:in `index'
Completed 200 OK in 36ms (Views: 14.4ms | ActiveRecord: 4.7ms | Allocations: 6583)


Started GET "/recipe" for ::1 at 2020-08-20 08:01:51 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category/1/recipe" for ::1 at 2020-08-20 08:02:00 -0400
  
ActionController::RoutingError (No route matches [GET] "/category/1/recipe"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category/1/recipes" for ::1 at 2020-08-20 08:02:03 -0400
Processing by RecipesController#index as */*
  Parameters: {"category_id"=>"1"}
  [1m[36mCategory Load (0.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
  [1m[36mRecipe Exists? (0.5ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:11:in `index'
  [1m[36mRecipe Load (0.7ms)[0m  [1m[34mSELECT "recipes".* FROM "recipes"[0m
  ↳ app/controllers/recipes_controller.rb:16:in `index'
Completed 200 OK in 109ms (Views: 91.5ms | ActiveRecord: 3.1ms | Allocations: 8453)


Started GET "/recipes" for ::1 at 2020-08-20 08:02:43 -0400
  
ActionController::RoutingError (No route matches [GET] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/recipes" for ::1 at 2020-08-20 08:03:50 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  
ActionController::RoutingError (No route matches [GET] "/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category/recipes" for ::1 at 2020-08-20 08:04:24 -0400
Processing by CategoryController#show as */*
  Parameters: {"id"=>"recipes"}
  [1m[36mCategory Load (0.4ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", nil], ["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:54:in `set_category'
Completed 404 Not Found in 25ms (ActiveRecord: 2.1ms | Allocations: 2232)


  
ActiveRecord::RecordNotFound (Couldn't find Category with 'id'=recipes):
  
app/controllers/category_controller.rb:54:in `set_category'
Started GET "/category/1/recipes" for ::1 at 2020-08-20 08:04:32 -0400
Processing by RecipesController#index as */*
  Parameters: {"category_id"=>"1"}
  [1m[36mCategory Load (0.5ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
  [1m[36mRecipe Exists? (2.0ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:11:in `index'
  [1m[36mRecipe Load (0.9ms)[0m  [1m[34mSELECT "recipes".* FROM "recipes"[0m
  ↳ app/controllers/recipes_controller.rb:16:in `index'
Completed 200 OK in 122ms (Views: 71.7ms | ActiveRecord: 4.7ms | Allocations: 10376)


Started DELETE "/category/1/recipes" for ::1 at 2020-08-20 08:06:15 -0400
  
ActionController::RoutingError (No route matches [DELETE] "/category/1/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started DELETE "/recipes/1" for ::1 at 2020-08-20 08:07:30 -0400
  
ActionController::RoutingError (No route matches [DELETE] "/recipes/1"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started DELETE "/category/1/recipes/1" for ::1 at 2020-08-20 08:07:37 -0400
Processing by RecipesController#destroy as */*
  Parameters: {"category_id"=>"1", "id"=>"1"}
  [1m[36mCategory Load (1.5ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
  [1m[36mRecipe Load (1.3ms)[0m  [1m[34mSELECT "recipes".* FROM "recipes" WHERE "recipes"."category_id" = ? AND "recipes"."id" = ? LIMIT ?[0m  [["category_id", 1], ["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:58:in `set_recipes'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
  [1m[36mRecipe Destroy (3.8ms)[0m  [1m[31mDELETE FROM "recipes" WHERE "recipes"."id" = ?[0m  [["id", 1]]
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
  [1m[35m (0.9ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
Completed 200 OK in 73ms (Views: 0.2ms | ActiveRecord: 7.7ms | Allocations: 6206)


Started DELETE "/category/1/recipes/2" for ::1 at 2020-08-20 08:08:13 -0400
Processing by RecipesController#destroy as */*
  Parameters: {"category_id"=>"1", "id"=>"2"}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
  [1m[36mRecipe Load (0.1ms)[0m  [1m[34mSELECT "recipes".* FROM "recipes" WHERE "recipes"."category_id" = ? AND "recipes"."id" = ? LIMIT ?[0m  [["category_id", 1], ["id", 2], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:58:in `set_recipes'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
  [1m[36mRecipe Destroy (0.5ms)[0m  [1m[31mDELETE FROM "recipes" WHERE "recipes"."id" = ?[0m  [["id", 2]]
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
  [1m[35m (3.7ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
Completed 200 OK in 11ms (Views: 0.2ms | ActiveRecord: 4.5ms | Allocations: 3293)


Started DELETE "/category/1/recipes/3" for ::1 at 2020-08-20 08:08:16 -0400
Processing by RecipesController#destroy as */*
  Parameters: {"category_id"=>"1", "id"=>"3"}
  [1m[36mCategory Load (0.1ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
  [1m[36mRecipe Load (0.3ms)[0m  [1m[34mSELECT "recipes".* FROM "recipes" WHERE "recipes"."category_id" = ? AND "recipes"."id" = ? LIMIT ?[0m  [["category_id", 1], ["id", 3], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:58:in `set_recipes'
Completed 404 Not Found in 6ms (ActiveRecord: 0.5ms | Allocations: 1463)


  
ActiveRecord::RecordNotFound (Couldn't find Recipe with [WHERE "recipes"."category_id" = ? AND "recipes"."id" = ?]):
  
app/controllers/recipes_controller.rb:58:in `set_recipes'
Started POST "/category/1/recipes" for ::1 at 2020-08-20 08:08:45 -0400
  [1m[35m (0.1ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
Processing by RecipesController#create as */*
  Parameters: {"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test", "category_id"=>"1", "recipe"=>{"name"=>"test", "ingredients"=>"test", "directions"=>"test", "notes"=>"test", "tags"=>"test"}}
  [1m[36mCategory Load (0.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
[31mUnpermitted parameter: :recipe[0m
  [1m[35m (0.4ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/recipes_controller.rb:26:in `create'
  [1m[36mRecipe Create (0.6ms)[0m  [1m[32mINSERT INTO "recipes" ("name", "ingredients", "directions", "notes", "tags", "category_id", "created_at", "updated_at") VALUES (?, ?, ?, ?, ?, ?, ?, ?)[0m  [["name", "test"], ["ingredients", "test"], ["directions", "test"], ["notes", "test"], ["tags", "test"], ["category_id", 1], ["created_at", "2020-08-20 12:08:45.760973"], ["updated_at", "2020-08-20 12:08:45.760973"]]
  ↳ app/controllers/recipes_controller.rb:26:in `create'
  [1m[35m (4.4ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/recipes_controller.rb:26:in `create'
Completed 500 Internal Server Error in 89ms (ActiveRecord: 7.7ms | Allocations: 12753)


  
NoMethodError (undefined method `name' for nil:NilClass):
  
app/controllers/recipes_controller.rb:27:in `create'
Started DELETE "/category/1/recipes/" for ::1 at 2020-08-20 08:11:19 -0400
  
ActionController::RoutingError (No route matches [DELETE] "/category/1/recipes"):
  
actionpack (6.0.3.2) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.3.2) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.3.2) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.3.2) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.3.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.3.2) lib/action_dispatch/middleware/host_authorization.rb:82:in `call'
railties (6.0.3.2) lib/rails/engine.rb:527:in `call'
puma (4.3.5) lib/puma/configuration.rb:228:in `call'
puma (4.3.5) lib/puma/server.rb:713:in `handle_request'
puma (4.3.5) lib/puma/server.rb:472:in `process_client'
puma (4.3.5) lib/puma/server.rb:328:in `block in run'
puma (4.3.5) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/category/1/recipes/" for ::1 at 2020-08-20 08:11:26 -0400
Processing by RecipesController#index as */*
  Parameters: {"category_id"=>"1"}
  [1m[36mCategory Load (1.3ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
  [1m[36mRecipe Exists? (0.6ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:11:in `index'
  [1m[36mRecipe Load (0.4ms)[0m  [1m[34mSELECT "recipes".* FROM "recipes"[0m
  ↳ app/controllers/recipes_controller.rb:16:in `index'
Completed 200 OK in 10ms (Views: 2.8ms | ActiveRecord: 2.3ms | Allocations: 2188)


Started DELETE "/category/1/recipes/3" for ::1 at 2020-08-20 08:12:08 -0400
Processing by RecipesController#destroy as */*
  Parameters: {"category_id"=>"1", "id"=>"3"}
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
  [1m[36mRecipe Load (0.3ms)[0m  [1m[34mSELECT "recipes".* FROM "recipes" WHERE "recipes"."category_id" = ? AND "recipes"."id" = ? LIMIT ?[0m  [["category_id", 1], ["id", 3], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:58:in `set_recipes'
  [1m[35m (0.1ms)[0m  [1m[36mbegin transaction[0m
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
  [1m[36mRecipe Destroy (2.5ms)[0m  [1m[31mDELETE FROM "recipes" WHERE "recipes"."id" = ?[0m  [["id", 3]]
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
  [1m[35m (1.4ms)[0m  [1m[36mcommit transaction[0m
  ↳ app/controllers/recipes_controller.rb:46:in `destroy'
Completed 200 OK in 18ms (Views: 0.5ms | ActiveRecord: 4.6ms | Allocations: 3301)


Started GET "/category/1/recipes" for ::1 at 2020-08-21 08:09:26 -0400
  [1m[35m (2.9ms)[0m  [1m[34mSELECT sqlite_version(*)[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
Processing by RecipesController#index as */*
  Parameters: {"category_id"=>"1"}
  [1m[36mCategory Load (0.9ms)[0m  [1m[34mSELECT "categories".* FROM "categories" WHERE "categories"."id" = ? LIMIT ?[0m  [["id", 1], ["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:62:in `set_category'
  [1m[36mRecipe Exists? (0.4ms)[0m  [1m[34mSELECT 1 AS one FROM "recipes" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/recipes_controller.rb:11:in `index'
Completed 200 OK in 45ms (Views: 0.4ms | ActiveRecord: 2.9ms | Allocations: 7731)


Started GET "/category" for ::1 at 2020-08-21 08:09:30 -0400
Processing by CategoryController#index as */*
  [1m[36mCategory Exists? (1.2ms)[0m  [1m[34mSELECT 1 AS one FROM "categories" LIMIT ?[0m  [["LIMIT", 1]]
  ↳ app/controllers/category_controller.rb:10:in `index'
  [1m[36mCategory Load (0.2ms)[0m  [1m[34mSELECT "categories".* FROM "categories"[0m
  ↳ app/controllers/category_controller.rb:15:in `index'
Completed 200 OK in 9ms (Views: 4.9ms | ActiveRecord: 1.4ms | Allocations: 1668)


  [1m[35m (0.8ms)[0m  [1m[35mCREATE DATABASE "recipes_development" ENCODING = 'unicode'[0m
  [1m[35m (0.5ms)[0m  [1m[35mCREATE DATABASE "recipes_test" ENCODING = 'unicode'[0m
  [1m[35m (0.3ms)[0m  [1m[34mSELECT pg_try_advisory_lock(8738300839714080705)[0m
  [1m[35m (1.7ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
  [1m[36mActiveRecord::InternalMetadata Load (1.1ms)[0m  [1m[34mSELECT "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2[0m  [["key", "environment"], ["LIMIT", 1]]
  [1m[35m (0.3ms)[0m  [1m[34mSELECT pg_advisory_unlock(8738300839714080705)[0m
  [1m[35m (0.8ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
